test/functional/apps/management/_getting_started.js

```diff
@@ -0,0 +1,59 @@
+import expect from 'expect.js';
+
+export default ({ getService, getPageObjects }) => {
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+  const log = getService('log');
+
+  const PageObjects = getPageObjects(['common', 'gettingStarted']);
+
+  describe('Getting Started page', () => {
+    describe('when no index patterns exist', () => {
+      beforeEach(async () => {
+        // delete .kibana index and then wait for Kibana to re-create it
+        await kibanaServer.uiSettings.replace({});
+      });
+
+      describe('when user has not opted out of Getting Started page', () => {
```

Comment:
You might want to add a before hook here that ensures that the getting started local storage entry is not set, with `remote.deleteLocalStorageItem()` https://theintern.github.io/leadfoot/module-leadfoot_Command.html#deleteLocalStorageItem